android/src/main/java/com/swmansion/gesturehandler/core/GestureUtils.kt

```diff
@@ -47,4 +53,74 @@ object GestureUtils {
   }
   fun coneToDeviation(angle: Double): Double =
     cos(Math.toRadians(angle / 2.0))
+
+  fun updateStylusData(event: MotionEvent): StylusData {
+    // On web and iOS 0 degrees means that stylus is parallel to the surface. On android this value will be PI / 2.
+    val altitudeAngle = (PI / 2) - event.getAxisValue(MotionEvent.AXIS_TILT).toDouble()
+    val pressure = event.getPressure(0).toDouble()
+
+    val orientation = event.getOrientation(0).toDouble()
+
+    // To get azimuth angle, we need to use orientation property (https://developer.android.com/develop/ui/compose/touch-input/stylus-input/advanced-stylus-features#orientation).
+    val azimuthAngle = (orientation + PI / 2).mod(2 * PI)
+
+    val tilts = spherical2tilt(altitudeAngle, azimuthAngle)
```

Comment:
I think there's no need for blank lines

Reply:
Changed while moving it to static function ([6589a60](https://github.com/software-mansion/react-native-gesture-handler/pull/3111/commits/6589a60db35aaad6aa67572adb968654a918a459))